module/applications/actor/base-sheet.mjs

```diff
@@ -1165,6 +1168,23 @@ export default class ActorSheet5e extends ActorSheet {
 
   /* -------------------------------------------- */
 
+  /**
+   * Change the quantity of an Owned Item within the actor.
+   * @param {Event} event        The triggering click event.
+   * @returns {Promise<Item5e>}  Updated item.
+   * @private
```

Comment:
This should be `@protected`. So should the two methods above it too, now that I look at it.